api/python/provisioner/commands/validate_setup.py

```diff
@@ -0,0 +1,181 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import logging
+from typing import List
+
+from ..inputs import (
+    ClusterParams,
+    StorageParams,
+    NodeParams
+)
+from ..vendor import attr
+from ..config import (
+    NODE_DEFAULT,
+    STORAGE_DEFAULT
+)
+
+from ..values import UNCHANGED
+
+logger = logging.getLogger(__name__)
+
+
+@attr.s(auto_attribs=True)
+class ValidateSetup:
```

Comment:
It's not a command

I believe we don't need that module inside `commands`.  And that class with the only static method looks unnecessary.
So I suggest:
- move the method (the logic of) `_parse_nodes` to `ConfigureSetup`
- move the module outside `commands` since there would be actually definitions of inputs for configuration file

Reply:
This was originally to validate node sections and count - which I later split to PR 910. 
We'll move some common modules to outside of `commands/` to `utils/` which will be addressed in a separate ticket - https://jts.seagate.com/browse/EOS-18518 .

Reply:
As I see that module is a new one and came from inputs.py which is outside the  `commands` directory. And EOS-18518 is about restructuring **inside** commands. Currently I think we don't have any common modules inside commands.

Reply:
Agreed! So, for now, I've moved it out of `commands/`. Will handle other alterations in future scope. Thanks